lib/parallel-api.js

```diff
@@ -130,7 +130,13 @@ function serializeOptions(options) {
 }
 
 function transformString(string, options) {
-  if (JOBS > 1 && transformIsParallelizable(options)) {
+  const isParallelizable = transformIsParallelizable(options);
+
+  if (options.throwUnlessParallelizable && !isParallelizable) {
+    throw new Error('BroccoliBabelTranspiler was configured to `throwUnlessParallelizable` and was unable to parallelize an plugin. Please see: https://github.com/babel/broccoli-babel-transpiler#parallel-transpilation for more details');
```

Comment:
It would generally be nice to explain _why_ it isn’t parallelizable. Not a deal breaker, but it would be much easier to help track these things down in the future...

Reply:
I would prefer to add the "why" to the readme section. Ideally we could say which plugin it was, but I don't know if we can reasonably infer the name...